src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj

```diff
@@ -1001,6 +1001,9 @@
     <None Include="ConverterTestData\ClangAnalyzer\RealLog.xml">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="DirectProducerTestData\Roslyn\RoslynLog.sarif">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
```

Comment:
It does and I'll do that. I wanted to at least save this as a stress test. We can just leave the branch behind.

I also want to talk about how poorly the validation scales.





On Thu, May 19, 2016 at 7:19 AM -0700, "Larry Golding" <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj<https://github.com/Microsoft/sarif-sdk/pull/289#discussion_r63886537>:

> @@ -1001,6 +1001,9 @@
>      <None Include="ConverterTestData\ClangAnalyzer\RealLog.xml">
>        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
>      </None>
> +    <None Include="DirectProducerTestData\Roslyn\RoslynLog.sarif">
> +      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
> +    </None>


Does this mean that every time I run BuildAndTest.cmd, it will take 3 minutes longer to run? I suggest that instead you generate a much smaller Roslyn log to add to the sarif-sdk.

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/sarif-sdk/pull/289/files/cab5a0d228110f4a92799b04d52ab9faaf25a08e#r63886537>